### ISSUE TITLE
delegate functions not getting called for `native` implementation

### ISSUE THREAD

**Issue by ajkerfaisal** [ivanvorobei/SPPermissions#260]
Delegate functions are not getting called for `native` implementation

```
let controller = SPPermissions.native([.camera])
controller.delegate = self
controller.present(on: vc)
```

But when implementation is for dialog, delegates are getting fired.
```
let controller = SPPermissions.dialog([.camera])
controller.delegate = self
controller.present(on: vc)
```

What I am missing here?

**Comment by ivanvorobei** [comment]
Can you save controller as property? Problem this, after present controller removed from memory by system and delegate don't available because week.

**Comment by ivanvorobei** [comment]
Fixed in version `6.6.5`. Please, update.
